examples/osggeometry/osggeometry.cpp

```diff
@@ -543,9 +545,10 @@ class MyTransformCallback : public osg::NodeCallback
 
         float               _angular_velocity;
 
-};
-
+}; //class MyTransformCallback
 
+/// Create a quad that sits behind the geometry from createScene()
+/// and shows the image from the OpenGL docs that geometry duplicates.
```

Comment:
filnet, I was looking at the [doxygen manual](https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html#cppblock) recently and noticed that `///` works the same as `/**` to introduce a documentation block.  My personal style is to avoid `/* */` so that I don't have any nested-comment issues, so I decided to use `///` going forward in my own code.  If you prefer `/**`, just let me know --- I'm happy to change it.